automatic_data_generation/train.py

```diff
@@ -323,7 +325,7 @@ def train(model, datasets, args):
         counter = 0
         import random
         while counter < args.datasize:
-            row = random.choice(train_reader)
+            row = random.choice(train_reader[1:]) # skip the header
```

Comment:
Warning: PTB does not have a header, the skip header will be an argument of the Dataset class in the new implem